README.md

```diff
@@ -39,75 +37,6 @@ Each package has its own **node_modules** directory that contains symbolic links
 
 > See [supported platforms](./docs/learning/SupportedPlatforms.md) for further information.
 
-## Build Instructions
-
-1. Clone repository (first time) with `git clone` or pull updates to the repository (subsequent times) with `git pull`
```

Comment:
I still feel like there is value in having the very basic build instructions included in the README, They are short enough to not clutter the page, but also functions as a 'quick start'.

Reply:
ahh, a quick start makes sense... is there anything 'visual' we could add as part of the quick start? can DTA be run without any envs?

Reply:
I agree that something visual or showcasey would be excellent, but I think we should include a new 'Quick start section' that contains an actual example for *using* itwin.

 For the Contributing Guidelines section, which is developer focused, I feel like having a very basic set of build instructions like we have now + a link to the full guidelines and what can be found there would be great.
 
 What I mean to say is, I think there should be both a QuickStart section for users, and a "mini startup" under the contribution guide lines as well for developers. Thoughts?

Reply:
Okay, I added a quick start section at the top, that has links to a viewer sample in our sandbox, and a link to other samples (in developer.bentley.com)